app/scripts/templates/settings/account_recovery/confirm_revoke.mustache

```diff
@@ -0,0 +1,15 @@
+<div id="account-recovery-confirm-revoke">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <form novalidate>
+            <h3>{{#t}}Are you sure you want to revoke?{{/t}}</h3>
+            <p class="revoke-description">{{#t}}You will be unable to use your recovery key to access your data in the event you reset your password.{{/t}}</p>
+
+            <div class="button-row">
+                <button action="submit" class="settings-button warning-button revoke">{{#t}}Revoke{{/t}}</button>
+                <button class="settings-button secondary-button cancel-link">{{#t}}Cancel{{/t}}</button>
```

Comment:
The link here looks very out of place. Other panels use a grey secondary button, e.g., setting an avatar via the camera:

<img width="411" alt="screen shot 2018-07-31 at 16 36 16" src="https://user-images.githubusercontent.com/848085/43470164-dc26abca-94df-11e8-8104-90cf24a9b66a.png">
<img width="426" alt="screen shot 2018-07-31 at 16 31 13" src="https://user-images.githubusercontent.com/848085/43470168-de4483a0-94df-11e8-9f6d-195c7df21441.png">

Delete account uses a grey secondary button too:

<img width="503" alt="screen shot 2018-07-31 at 15 47 06" src="https://user-images.githubusercontent.com/848085/43470211-f10510fe-94df-11e8-985c-f31a59d5ded2.png">